io.asgardio.tomcat.saml.agent.sample/pom.xml

```diff
@@ -22,17 +22,16 @@
         <groupId>io.asgardio.tomcat.saml.agent</groupId>
         <artifactId>tomcat-saml-agent</artifactId>
         <version>0.1.1-SNAPSHOT</version>
-        <relativePath>../../pom.xml</relativePath>
+        <relativePath>../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>SampleApp</artifactId>
+    <artifactId>io.asgardio.tomcat.saml.agent.sample</artifactId>
     <packaging>war</packaging>
 
     <build>
-        <finalName>SampleApp</finalName>
+        <finalName>io.asgardio.tomcat.saml.agent.sample</finalName>
```

Comment:
Shall we have the application name as `sample-app`.

Current name is too long and format doesn't look nice for the URL once app is deployed.

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/7bc0a7b510c5137c156f5317497b12e794ef3993